README.md

```diff
@@ -20,9 +20,19 @@ Once Helm is set up properly, add the repo as follows:
 helm repo add trino https://trinodb.github.io/charts/
 ```
 
-You can then run `helm search repo trino` to see the charts.
+Run `helm search repo trino` to see the latest charts in the `trino` repo with
```

Comment:
It searches for `trino` in all charts in all locally installed repositories. You'd get the output below only if there are no other charts from other repos starting with this prefix.

Reply:
Yeah .. I suppose.. let me reword